dart/lib/src/protocol/sentry_log.dart

```diff
@@ -0,0 +1,33 @@
+import 'sentry_id.dart';
+import 'sentry_log_level.dart';
+import 'sentry_log_attribute.dart';
+
+class SentryLog {
+  DateTime timestamp;
+  SentryId traceId;
+  SentryLogLevel level;
+  String body;
+  Map<String, SentryLogAttribute> attributes;
+  int? severityNumber;
+
+  SentryLog({
+    required this.timestamp,
+    required this.traceId,
+    required this.level,
+    required this.body,
+    required this.attributes,
+    this.severityNumber,
+  });
+
+  Map<String, dynamic> toJson() {
+    return {
+      'timestamp': timestamp.toIso8601String(),
+      'trace_id': traceId.toString(),
+      'level': level.value,
+      'body': body,
+      'attributes':
+          attributes.map((key, value) => MapEntry(key, value.toJson())),
+      if (severityNumber != null) 'severity_number': severityNumber,
```

Comment:
one last thing, if it is null we need to infer the value like so:

https://github.com/getsentry/sentry-javascript/blob/caa13167c5d6e35dd2a5dbc08b86f0f0f40681eb/packages/core/src/logs/constants.ts#L8